src/implementations/vanilla/components/select/select.js

```diff
@@ -524,6 +531,29 @@ export class Select {
       this.searchContainer.style.display = 'none';
     }
   }
+
+  /**
+   * Reset Multiselect.
+   */
+  resetForm() {
+    // A slight timeout is necessary to execute the function just after the original reset of the form.
+    setTimeout(() => {
+      Array.from(this.select.options).forEach((option) => {
```

Comment:
but.. i wonder, shouldn't actually be an autoInit().update operation the one that we should perform on form reset..? Or destroy and init, probably better..? 
Currently we have an issue:
![image](https://user-images.githubusercontent.com/14326885/166236809-6d58beec-d4d9-4eed-9ab4-dbe2bbc40cc0.png)
but this made me think that a "reset" event on a form might actually be one use case for a destroy method for our components.. :)

Reply:
What's the issue? 
it's normal that Bulgaria is selected after reset, as it is since the beginning? It reset the form element to the default states, and Bulgaria is selected by default.

Reply:
the issue is that the label Bulgaria disappears from the select, like if there is no option selected

Reply:
Strange it works just fine locally here, can you ensure JS is fully recompile?